tests/cephfs/cephfs_system/mds_standby_replay_node_failure_ops.py

```diff
@@ -0,0 +1,426 @@
+                    standby_replay_mds = out_iter.split()[2]
+                    mds_pair_info[mds["rank"]].update({active_mds: standby_replay_mds})
+    log.info(f"MDS Standby Replay pair info : {mds_pair_info}")
+    return mds_pair_info
+
+
+def validate_standby_replay_mds(client, fs_name, initial_mds_config, initial_mds_pair):
+    mds_config = get_mds_config(client, fs_name)
+    initial_standby_replay_mds = {}
+    for mds in initial_mds_config:
+        if mds.get("state") == "standby-replay":
+            initial_standby_replay_mds.update({mds["name"]: {"events": mds["events"]}})
+    log.info(f"initial_standby_replay_mds:{initial_standby_replay_mds}")
+    standby_replay_mds = {}
+    for mds in mds_config:
+        if mds.get("state") == "standby-replay":
+            standby_replay_mds.update({mds["name"]: {"events": mds["events"]}})
+    log.info(f"standby_replay_mds:{standby_replay_mds}")
+    log.info(f"initial_standby_replay_mds.keys():{initial_standby_replay_mds.keys()}")
+    log.info(f"standby_replay_mds.keys():{standby_replay_mds.keys()}")
+    if len(initial_standby_replay_mds.keys()) == len(standby_replay_mds.keys()):
+        log.info("Standby-Replay MDS count before and after test are same")
+    else:
+        str = f"Before : {initial_standby_replay_mds}, After : {standby_replay_mds}"
+        log.error(f"StandbyReplay MDS count before and after is not same,{str}")
+        return 1
+    sleep(5)
+    mds_config1 = get_mds_config(client, fs_name)
+    standby_replay_mds1 = {}
+    mds_pair = get_mds_standby_replay_pair(client, fs_name, mds_config)
+    log.info(f"mds_pair before:{initial_mds_pair},mds_pair after: {mds_pair}")
+    for mds in mds_config1:
+        if mds.get("state") == "standby-replay":
+            standby_replay_mds1.update({mds["name"]: {"events": mds["events"]}})
+            log.info(f"Events before:{standby_replay_mds[mds['name']]['events']}")
+            log.info(f"Events now:{standby_replay_mds1[mds['name']]['events']}")
+            if (
+                standby_replay_mds1[mds["name"]]["events"]
+                != standby_replay_mds[mds["name"]]["events"]
+            ):
+                log.info(f"StandbyReplay MDS {mds['name']} is operational")
+                return 0
+    return 1
+
+
+def wait_for_healthy_ceph(client1, fs_util, wait_time_secs):
+    # Returns 1 if healthy, 0 if unhealthy
+    ceph_healthy = 0
+    end_time = datetime.now() + timedelta(seconds=wait_time_secs)
+    while ceph_healthy == 0 and (datetime.now() < end_time):
+        try:
+            fs_util.get_ceph_health_status(client1)
+            ceph_healthy = 1
+        except Exception as ex:
+            log.info(ex)
+            log.info(
+                f"Wait for sometime to check if Cluster health can be OK, current state : {ex}"
+            )
+            sleep(5)
+    if ceph_healthy == 0:
+        return 0
+    return 1
+
+
+def start_io_time(fs_util, client1, mounting_dir, timeout=300):
```

Comment:
This has been moved to utilsV1, please reuse it here

Reply:
I tried to use the one in fsutilsV1, but there is a failure, this could be due to dependency on global variable within script whose change in value cant be passed down to other file dynamically which is fsutilsV1.

Also, i have not seen any test script using start_io_time from fsutilsV1, so i doubt if the one in fsutilsV1 is in usable mode. So i cant incorporate this comment.

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-97S7AT/mds_standby_replay_node_failure_ops_0.log 
File "/home/sumr/cg_quiesce/cephci/tests/cephfs/cephfs_utilsV1.py", line 4609, in start_io_time
    if stop_flag:
NameError: name 'stop_flag' is not defined

Reply:
I am using them in  "def runio_modify_max_mds"    &  "def runio_reboot_s_replay_mds_nodes" for multimds functional tests
That's okay for now. Maybe in the next PR you can make changes